Automattic/newspack-newsletters
# Add email validation

### All Submissions:






### Changes proposed in this Pull Request:



Adds email validation in editor.

### How to test the changes in this Pull Request:

1. Add or edit a newsletter 
3. Observe email is validated* in two email inputs:
    1. In sidebar "Newsletter Settings" section  in "From"->"Email" input
    1. in test email sending email input

\* it's not meant as a 100% email validation, just a convenience for the user, so they don't e.g. send a test email to an empty email

### Other information:
